_i18n/es/get-started/using.md

```diff
@@ -27,24 +27,23 @@
                 <div class="info-block">
                     <div class="row center-xs">
                         <div class="col">
-                            <h2>2. Request Support</h2>
+                            <h2>2. Solicita Soporte</h2>
                         </div>
                     </div>
                     <div class="row start-xs">
-                        <p>There is a large and supportive community that will assist if you experience any difficulty. See the <a href="{{site.baseurl}}/community/hangouts/">Hangouts</a> page for more information.</p>
+                        <p>Existe una gran comunidad de soporte que te ayudará en caso de experimentar cualquier dificultad. Accede a esta página para ampliar información <a href="{{site.baseurl}}/community/hangouts/">Hangouts</a>.</p>
                     </div>
                 </div>
             </div>
             <div class="right half col-lg-6 col-md-6 col-sm-12 col-xs-12">
                 <div class="info-block">
                     <div class="row center-xs">
                         <div class="col">
-                            <h2>3. Generate a Wallet</h2>
+                            <h2>3. Crea una cartera (Wallet)</h2>
```

Comment:
"monedero" term would be my choice too, but I opted for "cartera" because phrases like "el monedero de monero" could be confusing as both word are very similar.

Reply:
Yes understand....mmm...What do you think if we say "Monedero"? Take out "de monero". Just a suggestion :). I have thought about it too...